Code Cleanup

mcdonnnj opened the pull request:
# <!-- Use the title to describe PR changes in the imperative mood --> #

## 🗣 Description ##

This PR is mostly just some housekeeping for this project. There is a bug fixed in 93798d3, and a potential side-effect is removed in 28c9247, but the bulk is some general polish. The logging has also been adjusted to `printf` style that is preferred because of `logging` library internals.
<!-- Describe the "what" of your changes in detail. -->
<!-- To avoid scope creep, limit changes to a single goal. -->

## 💭 Motivation and context ##

The bug I mentioned is a problem for anyone else trying to use this code in production. While I was fixing the aforementioned bug, I wanted to do some general housekeeping since this project was created before we moved to an "Initial Pull Request" model as talked about in the [development guide](https://github.com/cisagov/development-guide/tree/develop/project_setup#create-an-initial-pull-request).
<!-- Why is this change required? -->
<!-- What problem does this change solve? How did you solve it? -->
<!-- Mention any related issue(s) here using appropriate keywords such
<!-- as "closes" or "resolves" to auto-close them on merge. -->

## 🧪 Testing ##

Automated testing passes and some basic checks I ran locally gave expected results.
<!-- How did you test your changes? How could someone else test this PR? -->

<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->

## ✅ Checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- Draft PRs should have one or more unchecked boxes. -->
<!-- If you're unsure about any of these, don't hesitate to ask. -->
<!-- We're here to help! -->

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

mcdonnnj committed:
Add a default value for a message output

There was the possibllity of falling through to an unset variable previously.

mcdonnnj committed:
Add some missing return value type hints

Type hints should be added whenever possible. This fixes missing return value
hints for two methods.

mcdonnnj committed:
Remove parameter deletion logic

Deleting the parameter when no users are present may cause dependent tools to
fail instead of performing a no-op because it is empty.

mcdonnnj committed:
Sort the members of the operator list parameter

Sorting the users that are turned into a comma delimited list makes it easier
for manual interaction in the AWS console. The tool should be preferred, but we
should still consider other interactions.

mcdonnnj committed:
Simplify return statements

The ManageOperators.add_user() and ManageOperators.remove_user() methods were
needlessly complicated about the return value. This simplifies them since the
method they are relying on for functionality returns the same type of value.

mcdonnnj committed:
Update logging in manageoperators.py

Replace all of the f-strings with the substitution based log messaging that is
preferred.

mcdonnnj committed:
Use 'err' for caught exception objects

Single character variable names are generally discouraged for Python.

mcdonnnj committed:
Simplify some code in cli.py

Simply of an over-complicated check in Schema validation. Remove an unnecessary
else return usage.

dav3r commented on src/manage_cyhy_ops/manageoperators.py:
```diff
@@ -36,94 +36,82 @@ def _get_cyhy_ops_list(self):
+        try:
+            # The SSM response on success currently only contains a version
+            # number and the parameter tier.
+            # Neither are useful to us at this time, so we don't store them..
```
Is the double-period intentional here?  I saw another one elsewhere.

```suggestion
            # Neither are useful to us at this time, so we don't store them.
```

mcdonnnj replied:
Nope, nice catch. Fixed both instances in 5b1711c.

dav3r reviewed (approved):
Nice improvements! 🎟️

mcdonnnj committed:
Fix double period typos

It was pointed out in review that I had some unintentional double periods, so I
have fixed them.
